lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+    bernardine_of_siena_priest: 'சியன்னா நகர்ப் புனித பெர்னார்தின் - அருள்பணியாளர்',
+    beuno_of_wales_abbot: 'வேல்சின் புனித பியுனோ - மடாதிபதி',
+    blaise_of_sebaste_bishop: 'புனித ப்ளேஸ் - ஆயர், மறைச்சாட்சி',
+    blessed_martyrs_of_paris: 'பாரிஸ் நகரின் முத்திப்பேறுபெற்ற வேதசாட்சிகள் - மறைச்சாட்சியர்',
+    bogumilus_of_dobrow_bishop: 'முத்திப்பேறுபெற்ற போகுமிலஸ் - ஆயர்',
+    boleslawa_mary_lament_virgin: 'முத்திப்பேறுபெற்ற போலெஸ்லவா மரிய லமெண்ட் - கன்னி',
+    bonaventure_of_bagnoregio_bishop: 'புனித பொனவென்சர் - ஆயர், மறைவல்லுநர்',
+    boniface_of_mainz_bishop: 'புனித பொனிபஸ் - ஆயர், மறைசாட்சி',
+    boris_of_kiev_and_gleb_of_kiev_martyrs: 'புனித பொறிஸ், புனித க்ளெப் - மறைசாட்சிகள்',
+    brendan_of_clonfert_abbot: 'புனித பிறெண்டன் - மடாதிபதி',
+    bridget_of_sweden_religious: 'புனித பிரிட்ஜிட் - துறவி',
+    bridget_of_sweden_religious_copatroness_of_europe: 'புனித பிரிட்ஜிட் - துறவி, ஐரோப்பாவின் பாதுகாவலி',
+    brigid_of_kildare_virgin: 'புனித பிரிட்ஜிட் - மடாதிபத',
+    brigid_of_kildare_virgin_copatroness_of_ireland: 'புனித பிரிட்ஜிட் - கன்னி, அயர்லாந்தின் பாதுகாவலி',
+    bronislava_of_poland_virgin: 'போலந்தின் முத்திப்பேறுபெற்ற ப்றோனிஸ்லாவா - கன்னி',
+    bronislaw_markiewicz_priest: 'முத்திப்பேறுபெற்ற பறோனிஸ்லோ மார்கவிச் - அருட்பணியாளர்',
+    bruno_of_cologne_priest: 'புனித புரூனோ - அருள்பணியாளர்',
+    bruno_of_querfurt_bishop: 'கியூபோர்டின் புனித புரூனோ - ஆயர், மறைசாட்சி',
+    caesarius_of_arles_bishop: 'ஆர்ல்சின் புனித சிசேரியஸ் - ஆயர்',
+    cajetan_of_thiene_priest: 'புனித கஜெடான் - அருட்பணியாளர்',
+    callistus_i_pope: 'புனித முதலாம் கலிஸ்டஸ் - திருத்தந்தை, மறைசாட்சி',
+    camillus_de_lellis_priest: 'புனித கமிலஸ் டி லெலிஸ் - அருட்பணியாளர்',
+    canice_of_kilkenny_abbot: 'புனித கனிஸ் - மடாதிபதி',
+    canute_iv_of_denmark_eric_ix_of_sweden_and_olaf_ii_of_norway_martyrs:
+      'புனிதர்கள் கனுடே, எறிக், ஒலெப் - மறைசாட்சிகள்',
+    canute_iv_of_denmark_martyr: 'புனிதர் கனுடே - மறைசாட்சி',
+    carlos_manuel_rodriguez_santiago: 'முத்திப்பேறுபெற்ற கார்லோஸ் மானுவேல் றொட்றிகூஸ் சந்தியாகு',
+    carmelites_of_compiegne_virgins_and_martyrs: 'கம்பீனின் முத்திப்பேறுபெற்ற கார்மலைட் - கன்னியர்கள், மறைசாட்சிகள்',
+    caroline_kozka_virgin: 'முத்திப்பேறுபெற்ற கரோலின் கொஸ்கா - கன்னி, மறைசாட்சி',
+    carthage_of_lismore_bishop: 'புனித கார்தேஜ் - ஆயர்',
+    casimir_of_poland: 'போலந்தின் புனித காஸ்மிர்',
+    catherine_of_alexandria_virgin: 'அலெக்சான்றியாவின் புனித கத்தரின் - கன்னி, மறைசாட்சி',
+    catherine_of_saint_augustine_de_simon_de_longpre_virgin: 'புனித அகுஸ்தினின் புனித கத்தரின் - கன்னி',
+    catherine_of_siena_virgin: 'சியன்னாவின் புனித கத்தரின் - கன்னி, மறைவல்லுநர்',
+    catherine_of_siena_virgin_copatroness_of_europe:
+      'சியன்னாவின் புனித கத்தரின் - கன்னி, மறைவல்லுநர், இத்தாலி, ஐரோப்பவின் பாதுகாவலி',
+    catherine_of_siena_virgin_copatroness_of_italy_and_europe:
+      'சியன்னாவின் புனித கத்தரின் - கன்னி, இத்தாலி, ஐரோப்பவின் பாதுகாவலி',
```

Comment:
According to Google Translate, the value of both keys is (nearly, apart of `Saint,`) the same:

- `சியன்னாவின் புனித கத்தரின் - கன்னி, மறைவல்லுநர், இத்தாலி, ஐரோப்பவின் பாதுகாவலி` → `St. Catherine of Sienna - Virgin, Saint, Protectress of Italy, Europe`;
- `சியன்னாவின் புனித கத்தரின் - கன்னி, இத்தாலி, ஐரோப்பவின் பாதுகாவலி` → `Saint Catherine of Sienna - Virgin, Protectress of Italy, Europe`.

Also note that `catherine_of_siena_virgin_copatroness_of_europe` is a key used for all European countries except for Italy where there is St Catherine of Siena celebrated as _Copatroness of Europe_, while `catherine_of_siena_virgin_copatroness_of_italy_and_europe` is used in Italy only, where she is celebrated as a _Copatroness_ of both _Italy and Europe_.

Could you verify the key values, please?